.github/workflows/build-library.yml

```diff
@@ -28,15 +28,20 @@ jobs:
         python-version:
           - setup: '3.8'
             tox: 'py38'
+            grpc-port: 10085
```

Comment:
Different ports make sure the test matrix can run without conflict.

Reply:
No, you're right, each one gets its own worker - I thought it would be useful to have dedicated ports, but I could remove that part, it's not likely to run them in parallel locally.

Reply:
Removed now